packages/toi/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@toi/toi",
-  "version": "1.2.0",
```

Comment:
This is a minor bump, not patch. You've added new features (expanded the options object).